src/coord/src/catalog.rs

```diff
@@ -1268,6 +1268,16 @@ impl Catalog {
         &self.indexes
     }
 
+    /// Returns the default index for the specified `id`.
+    ///
+    /// Panics if `id` does not exist, or if `id` is not an object on which
+    /// indexes can be built.
+    pub fn default_index_for(&self, id: GlobalId) -> Option<GlobalId> {
```

Comment:
This function is much clearer, thanks!

Is the selection of the default index something we plan on improving later on?